tools/azure-sdk-tools/tests/integration/scenarios/pyproject_project_def_beta/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]  # Requires 61.0.0 for dynamic version
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "azure-keyvault-keys"
+authors = [
+    {name = "Microsoft Corporation", email = "azurekeyvault@microsoft.com"}
+    ]
+description = "Microsoft Azure Key Vault Keys Client Library for Python"
+requires-python = ">=3.7"
```

Comment:
Nit: this should be 3.8 based on the `classifiers`

Reply:
It's just test fixture so I wasn't looking at correctness there, but your point is well taken. Will update to be consistent.

Reply:
Updated.